app/views/shared/footer/_footer.html.erb

```diff
@@ -17,5 +17,22 @@
         <%= render 'shared/footer/footer_menu' %>
       </div>
     </div>
+    <% unless cookie_policy_accepted? %>
+      <div class="cookie-policy" data-dough-component="Cookie">
+        <%= form_tag(
+          cookie_policy_acceptances_path,
+          remote: true,
+          authenticity_token: true,
+          class: 'cookie-form' # hook class for js
+        ) do %>
+          <p class="cookie-policy-content">
+            <%= t('fincap.cookie_policy.body_html', url: t('fincap.cookie_policy.guide_link')) %>
+            <button class="btn btn--primary cookie-policy--close-button">
```

Comment:
I think this **button** shouldn't be inside the **p** tag, putting both on the same level should solve my previous comment.

Reply:
@andreferreiradlw The problem with that is that the `<button>` is always pushed under the `<p>`, even when the text is not full width. I think it's OK to leave them as they are and just change the margins.